src/CMakeLists.txt

```diff
@@ -587,15 +585,22 @@ if(${WITH_RADOSGW})
     cls_statelog_client
     cls_rgw_client
     fcgi
-    ${CURL_LIBRARIES}
     ${EXPAT_LIBRARIES}
   )
 endif(${WITH_RADOSGW})
-if(${WITH_RBD})
+if(WITH_RBD)
   set(DENCODER_EXTRALIBS
       ${DENCODER_EXTRALIBS}
       rbd_replay_types)
-endif(${WITH_RBD})
+  if(LINUX)
+    list(APPEND dencoder_srcs
+        $<TARGET_OBJECTS:krbd_objs>)
+  endif(LINUX)
+endif(WITH_RBD)
+if(WITH_KVS)
+  list(APPEND dencoder_srcs
+      $<TARGET_OBJECTS:parse_secret_objs>)
```

Comment:
@tchaikov 
It is like you said: We discussed it before, I looked into it, and like above the include is obvious.
But it could very well be that we discussed this in private Email.

I think I've found ways to actually work around this, but that requires fixing a few source files.
So I'd rather do that after I'm finished with the Cmake stuff.